how-it-works/6_webaccess/01_web_access.subpage.md

```diff
@@ -7,55 +7,41 @@ slug: smart-contracts-serve-the-web
 
 # Smart Contracts serve the web
```

Comment:
```suggestion
```

Reply:
I don't understand this change: Should we remove the title of the page?

Reply:
The title is already above, there is no point in having it appear twice
![image](https://github.com/dfinity/portal/assets/46541899/d2f14eb6-d624-43e0-8ab7-ce52568c3cec)